packages/client/src/MWPClient.ts

```diff
@@ -119,6 +123,20 @@ export class MWPClient {
     return accounts;
   }
 
+  /**
+   * Request a nonce for origin verification
+   * @returns Promise<string> - The nonce provided by the wallet
+   */
+  async getNonce(): Promise<string> {
```

Comment:
hmm we probably don't need this func if we have `MWPClient.request({ method: 'wallet_getNonce' })`?